apps/service-catalog/components/basic-service-form/index.tsx

```diff
@@ -60,13 +65,17 @@ export const BasicServiceForm = ({ catalogId, service, type, statuses }: Service
       : createPublicService(catalogId, updatedService as ServiceToBeCreated);
   };
 
-  const handleDeleteService = () => {
+  async function handleDeleteService() {
```

Comment:
ikke feil, men for konsistens i koden. Tror vi har som mål å bruke kun arrow functions
```suggestion
  const handleDeleteService = async () => {
```

Reply:
Okei! Fikser på resten også 👌